image-builder/builder-services/src/main/java/io/hyscale/builder/services/impl/LocalImageBuildPushServiceImpl.java

```diff
@@ -49,6 +56,13 @@ public void buildAndPush(ServiceSpec serviceSpec, BuildContext context) throws H
         if (validate(serviceSpec) && isImageBuildPushRequired(serviceSpec, context)) {
             context = buildService.build(serviceSpec, context);
             pushService.pushImage(serviceSpec, context);
+    		String imagePullPolicy = System.getenv(IMAGE_PULL_POLICY);
```

Comment:
This should from the config itself & resolve the processor here from the factory class